proto/Cargo.toml

```diff
@@ -12,11 +12,11 @@ categories = ["network-programming"]
 build = "build.rs"
 
 [dependencies]
-protobuf = "~2.0"
+protobuf = "~2.2"
```

Comment:
Why isn't this tilde removed?

Reply:
Because CI uses `grpcio-proto`, we dont want to break CI due to third party crates.